src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+  const stylesArray = sourceString.split(';');
```

Comment:
[CODE KNOWLEDGE] - There's no need to create a separate array variable for the styles. You can chain the `split` and `forEach` methods directly to avoid unnecessary variable declarations.